pkg/local_object_storage/blobstor/common/get.go

```diff
@@ -15,3 +15,11 @@ type GetRes struct {
 	Object  *objectSDK.Object
 	RawData []byte
 }
+
+// GetBytesOptions groups options for [Storage.GetBytes].
+type GetBytesOptions struct {
```

Comment:
Please, let's not add more `...Options` if we're not talking about tens of them. Functions have arguments, that's what they're for.

Reply:
structured code is always better to me, seeing calls like `call(addr, nil)` makes you just imagine what it means. Required prms are explicit (1 in this case) 

so, i keep proposing this

Reply:
Is `call(addr, GetBytesOptions{})` better than `call(addr, nil)`?

Reply:
gone